source/dpp/runtime/context.d

```diff
@@ -217,7 +217,7 @@ struct Context {
        In C it's possible for a struct field name to have the same name as a struct
        because of elaborated names. We remember them here in case we need to fix them.
      */
-    void rememberField(in string spelling) @safe pure {
+    void rememberField(scope const string spelling) @safe pure {
```

Comment:
just a rant but i hate that they changed the meaning of `in`. oh well

Reply:
Did you see my [dmd PR](https://github.com/dlang/dmd/pull/10506)?